autoextract/__main__.py

```diff
@@ -116,6 +118,12 @@ def read_input(input_fp, intype, page_type):
                    choices=["DEBUG", "INFO", "WARNING", "ERROR"],
                    help="log level")
     p.add_argument("--shuffle", help="Shuffle input URLs", action="store_true")
+    p.add_argument("--disable-retries", action="store_true",
+                   help="Disable network, Request-level, and Query-level "
+                        "errors retrying.")
+    p.add_argument("--skip-query-errors", action="store_true",
```

Comment:
`skip-query-errors` doesn't reflect well what the flag is. Some ideas: `--dont-retry-on-query-errors` , `--never-retry-on-query-errors`, `--no-retry-on-query-errors` `--avoid-retrying-query-errors`, `--disable-query-errors-retrials`. 

Also probably the user of the library won't have any idea what "Query-level error" means at all. Probably we should clarify it somewhere. Maybe here is a good place.